File: planet-render/src/main/java/au/com/codeka/warworlds/planetrender/MainApplet.java
package au.com.codeka.warworlds.planetrender;

import javax.swing.JApplet;
import javax.swing.SwingUtilities;

public class MainApplet extends JApplet {
  private static final long serialVersionUID = 1L;

  /** Called when the applet is loaded into the browser. */
  public void init() {
    try {
      SwingUtilities.invokeAndWait(() -> add(new AppContent()));
    } catch (Exception e) {
      System.err.println("createGUI didn't complete successfully");
    }
  }
}

File: planet-render/src/main/kotlin/au/com/codeka/warworlds/planetrender/ui/MainApplet.kt
package au.com.codeka.warworlds.planetrender.ui

import javax.swing.JApplet
import javax.swing.SwingUtilities

class MainApplet : JApplet() {
  /** Called when the applet is loaded into the browser.  */
  override fun init() {
    try {
      SwingUtilities.invokeAndWait { add(AppContent()) }
    } catch (e: Exception) {
      System.err.println("createGUI didn't complete successfully")
    }
  }

  companion object {
    private const val serialVersionUID = 1L
  }
}